test/functional/comment_controller_test.rb

```diff
@@ -315,7 +315,7 @@ def teardown
           id: comment.id
     end
     assert_not_nil :answer
-    assert_not_equal initial_mail_count, ActionMailer::Base.deliveries.size
+    assert_equal initial_mail_count, ActionMailer::Base.deliveries.size
```

Comment:
oh, so this then wouldn't trigger a new notification, right? Maybe we should add a comment here explaining that? 

Thank you!

Reply:
yes, its test that no email is generated on the promotion of comment to answer.I will add a comment explaining this or I can remove this check what you say @jywarren?